app/services/current-recognition.js

```diff
@@ -54,8 +61,11 @@ export default class CurrentRecognitionService extends Service {
       this.recognitionModel.endTime =
         await recognition.validityPeriod.get('endTime');
       this.recognitionModel.legalResource = recognition.legalResource ?? null;
-      this.recognitionModel.awardedBy = await recognition.awardedBy;
-      this.selectedItem = await recognition.awardedBy;
+      this.recognitionModel.awardedBy = await recognition.awardedBy.get('name');
```

Comment:
Having to use `.get()` is an indicator that the async relationship wasn't resolved before accessing it.

Something like this should work I think:
```suggestion
      this.recognitionModel.awardedBy = (await recognition.awardedBy).name;
```